docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,242 @@
+        security.protocol=SASL_SSL
+        sasl.mechanism=PLAIN
+        sasl.jaas.config=\
+            org.apache.kafka.common.security.plain.PlainLoginModule required \
+            username="<confluent cloud access key>" \
+            password="<confluent cloud secret>";
+
+    **Monitoring Interceptors for Control Center Streams Monitoring**
+
+    In this example, KSQL is configured to connect to an Apache Kafka cluster using SASL and SSL and integrate with |c3-short|
+    using the :ref:`Control Center interceptors <controlcenter_clients>`.
+
+
+    .. code:: bash
+
+        # Producer Confluent Monitoring Interceptors for Control Center streams monitoring
+        producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
+        producer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        producer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        producer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        producer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        producer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        producer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+        # Consumer Confluent Monitoring Interceptors for Control Center streams monitoring
+        consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+        consumer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        consumer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        consumer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        consumer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For more information, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+-----------------------------------------
+Configuration Example for Schema Registry
```

Comment:
I'd rename to: "Using KSQL with a secured Confluent Schema Registry"

Reply:
this section isn't about "using" - it's about configuring

Reply:
"Configuring KSQL for a secure Confluent Schema Registry"??